src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -706,6 +711,42 @@ def use_mutator_plugin(target_name, extra_env, chroot):
   return True
 
 
+def get_merge_directory():
+  """Returns the path of the directory we can use for merging."""
+  temp_dir = fuzzer_utils.get_temp_dir()
+  return os.path.join(temp_dir, MERGE_DIRECTORY_NAME)
+
+
+def create_merge_directory():
+  """Create the merge directory and return its path."""
+  merge_directory_path = get_merge_directory()
+  shell.create_directory(
+      merge_directory_path, create_intermediates=True, recreate=True)
+  return merge_directory_path
+
+
+def is_sha1_hash(possible_hash):
+  """Returns True if |possible_hash| looks like a valid sha1 hash."""
+  if len(possible_hash) != 40:
+    return False
+
+  return all(char in HEXDIGITS_SET for char in possible_hash)
+
+
+def move_mergeable_units(merge_directory, corpus_directory):
+  """Move new units in |merge_directory| into |corpus_directory|."""
+  initial_units = set(
+      os.path.basename(filename)
+      for filename in shell.get_files_list(corpus_directory))
+
+  for unit_path in shell.get_files_list(merge_directory):
+    unit_name = os.path.basename(unit_path)
+    if unit_name in initial_units and is_sha1_hash(unit_name):
```

Comment:
does anyone remember why `is_sha1_hash` check is needed? I can't understand how it helps